src/app/centers/createcenter/createcenter.component.html

```diff
@@ -0,0 +1,124 @@
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Staff</mat-label>
+            <mat-select formControlName="staffId">
+              <mat-option *ngFor="let staff of staffData" [value]="staff.id">
+                {{ staff.displayName }}
+              </mat-option>
+            </mat-select>
+
+          </mat-form-field>
+
+          <table>
+            <tr>
+              <td>
+                <mat-checkbox [formControl]="disableSelect" (click)="disable()">Active</mat-checkbox>
+              </td>
+              <td [hidden]="!disableSelect.value">
+                <mat-form-field class="example-full">
+                  <mat-label>Activation On</mat-label>
+                  <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="activationDateDatePicker"
+                    formControlName="activationDate" required >
+                  <mat-datepicker-toggle matSuffix [for]="activationDateDatePicker"></mat-datepicker-toggle>
+                  <mat-datepicker #activationDateDatePicker></mat-datepicker>
+                  <mat-error *ngIf="centerForm.controls.activationDate.hasError('required')">
+                    Activation Date is <strong>required</strong>
+                  </mat-error>
+                </mat-form-field>
+              </td>
+            </tr>
+          </table>
+
+          <mat-form-field>
+            <mat-label>External Id </mat-label>
+            <input matInput type="number" formControlName="externalId">
+          </mat-form-field>
+
+
+          <mat-form-field class="example-full">
+            <mat-label>Submitted On</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" formControlName="submittedOnDate"
+              >
+            <mat-datepicker-toggle matSuffix [for]="submittedOnDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #submittedOnDatePicker></mat-datepicker>
+            <mat-error *ngIf="centerForm.controls.submittedOnDate.hasError('required')">
+              Submitted On is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+          <table>
+            <tr>
+              <td>
+                <mat-form-field class="example-full">
+                  <mat-label>Select and add groups</mat-label>
+                  <mat-select>
+                    <mat-option *ngFor="let groups of groupData" [value]="groups.id" (click)="display(groups)">
+                      {{ groups.name }}
+                    </mat-option>
+                  </mat-select>
+                </mat-form-field>
+              </td>
+              <td [hidden]="value">
+                <mat-list *ngFor="let lst of groupDataDetail ">
+                  <button type="button" mat-mini-fab color="primary" (click)="add(lst)">  <fa-icon icon="plus"></fa-icon></button>
```

Comment:
The UI doesn't look good and is not responsive. Please update using flex layout directives and take reference from `create-journal-entries.component.html` in accounting for the way plus and minus buttons have been used. Also, ensure that once a group has been added I should not be able to add it again. Ensure the names of variables and function used are suitable and not `add ` or `lst` which do not tell us what are they being used for.